.github/CODEOWNERS

```diff
@@ -634,6 +634,7 @@
 #
 # On modifying this list make sure to keep the CODEOWNER_IMPORTANT_PATHS list
 # in scripts/linters/pre_commit_linter.py in sync with the modifications.
+/core/templates/services/svg-sanitizer.service.ts @BenHenning @seanlip
```

Comment:
@seanlip do you want to be included here?

Reply:
Yup I'm OK with that for now, thanks!